crates/genie-hki/Cargo.toml

```diff
@@ -11,3 +11,4 @@ repository = "https://github.com/SiegeEngineers/genie-rs"
 [dependencies]
 byteorder = "^1.3.1"
 flate2 = { version = "^1.0.0", features = ["rust_backend"], default-features = false }
+genie-lang = { path = "../genie-lang" }
```

Comment:
It needs a `version` as well as a `path` key for it to also be installable as an external crate with cargo (so it can download the correct version)